File: AppGenerateFiles/helpdesk/Model/CaseTable_Case.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class CaseTable_Case : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Case { get; set; }
       public string? Titulo { get; set; }
       public string? Descripcion { get; set; }
       public int? Id_Perfil { get; set; }
       public string? Estado { get; set; }
       public int? Id_Dependencia { get; set; }
       public DateTime? Fecha_Inicio { get; set; }
       public DateTime? Fecha_Final { get; set; }
       public int? Id_Servicio { get; set; }
       public int? Id_Vinculate { get; set; }
       public string? Mail { get; set; }
       public string? Case_Priority { get; set; }
       [ManyToOne(TableName = "CaseTable_VinculateCase", KeyColumn = "Id_Vinculate", ForeignKeyColumn = "Id_Vinculate")]
       public CaseTable_VinculateCase? CaseTable_VinculateCase { get; set; }
       [ManyToOne(TableName = "Cat_Dependencias", KeyColumn = "Id_Dependencia", ForeignKeyColumn = "Id_Dependencia")]
       public Cat_Dependencias? Cat_Dependencias { get; set; }
       [ManyToOne(TableName = "Tbl_Profile", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public Tbl_Profile? Tbl_Profile { get; set; }
       [ManyToOne(TableName = "Tbl_Servicios", KeyColumn = "Id_Servicio", ForeignKeyColumn = "Id_Servicio")]
       public Tbl_Servicios? Tbl_Servicios { get; set; }
       [OneToMany(TableName = "CaseTable_Comments", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public List<CaseTable_Comments>? CaseTable_Comments { get; set; }
       [OneToMany(TableName = "CaseTable_Mails", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public List<CaseTable_Mails>? CaseTable_Mails { get; set; }
       [OneToMany(TableName = "CaseTable_Tareas", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public List<CaseTable_Tareas>? CaseTable_Tareas { get; set; }
       [OneToMany(TableName = "Tbl_Profile_CasosAsignados", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public List<Tbl_Profile_CasosAsignados>? Tbl_Profile_CasosAsignados { get; set; }
   }
}

File: AppGenerateFiles/helpdesk/FrontModel/CaseTable_Case.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { CaseTable_VinculateCase }  from './CaseTable_VinculateCase.js'
import { Cat_Dependencias }  from './Cat_Dependencias.js'
import { Tbl_Profile }  from './Tbl_Profile.js'
import { Tbl_Servicios }  from './Tbl_Servicios.js'
import { CaseTable_Comments }  from './CaseTable_Comments.js'
import { CaseTable_Mails }  from './CaseTable_Mails.js'
import { CaseTable_Tareas }  from './CaseTable_Tareas.js'
import { Tbl_Profile_CasosAsignados }  from './Tbl_Profile_CasosAsignados.js'
class CaseTable_Case extends EntityClass {
   constructor(props) {
       super(props, 'EntityHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Case;
   /**@type {String}*/ Titulo;
   /**@type {String}*/ Descripcion;
   /**@type {String}*/ Estado;
   /**@type {Date}*/ Fecha_Inicio;
   /**@type {Date}*/ Fecha_Final;
   /**@type {String}*/ Mail;
   /**@type {String}*/ Case_Priority;
   /**@type {CaseTable_VinculateCase} ManyToOne*/ CaseTable_VinculateCase;
   /**@type {Cat_Dependencias} ManyToOne*/ Cat_Dependencias;
   /**@type {Tbl_Profile} ManyToOne*/ Tbl_Profile;
   /**@type {Tbl_Servicios} ManyToOne*/ Tbl_Servicios;
   /**@type {Array<CaseTable_Comments>} OneToMany*/ CaseTable_Comments;
   /**@type {Array<CaseTable_Mails>} OneToMany*/ CaseTable_Mails;
   /**@type {Array<CaseTable_Tareas>} OneToMany*/ CaseTable_Tareas;
   /**@type {Array<Tbl_Profile_CasosAsignados>} OneToMany*/ Tbl_Profile_CasosAsignados;
}
export { CaseTable_Case }
